### DISCUSSION TITLE
the program now opens in the browser

### DISCUSSION THREAD

**Discussion by dashadowofcat** [AssetRipper/AssetRipper#1280] [Q&A]
i used this program a few months ago and remember it being a normal application but now when i installed it again it opens on the browser

i may be downloading the wrong version or something like that, i would really appreciate help
thanks it advance!

**Comment by ds5678** [comment]
It's been like this since about 1.0.2.

**Reply by dashadowofcat** [reply]
ok thanks

**Comment by mextex** [comment]
Sorry for my stupid question: Im not able to run AssetRipper on my mac, as I dont know how. Could anyone give me a hint?
